module-2/backend/week_four.md

```diff
@@ -9,15 +9,15 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 
 ### Questions
 
-* What is a cookie?
-* What’s the difference between a session and a cookie?
-* What’s a flash and when do you want to use flashes?
-* Why do people say “HTTP is stateless”?
-* What’s authentication? Explain.
-* What’s the difference between authentication and authorization?
-* What’s a before filter?
-* How do we keep track of a user once they’ve logged in?
-* When do you want to namespace a resource? When do you want to nest a resource? What's the differences between those two approaches?
-* At a high level, what tools can you use to implement authorization? How would you use them?
-* What's an enum, and what advantages does it offer? What data type needs to be in your database to use an enum? Where do you declare an enum?
-* What are some strategies you can use to keep your views DRY?
+* What is a cookie? A cookie is a small hash of data that is created when a user visits (or does various things on) a particular webpage. 
+* What’s the difference between a session and a cookie? A session does not persist between visits, a.k.a. a session hash of data - which is similar to that of a cookie in the sense that it's stored as a small hash of data - will only be available while a user is logged into a page, and then the session will expire. Because of this, sessions are good for authentication. 
```

Comment:
a session persists until it's cleared on the server side. we decide when it expires 👍